conda/build.sh

```diff
@@ -22,11 +22,8 @@ fi
 # Show cmake settings
 cmake -B . -S .. -LA
 
-cmake --build . --target scipp-common-test -- -v 
```

Comment:
@DanNixon these could have been keeping the OSX job alive (see recent failure) https://travis-ci.org/scipp/scipp/jobs/648390797

Reply:
Looks like it, that does seem familiar now that I think about it.